src/IO/H5/StellarCollapseEos.hpp

```diff
@@ -0,0 +1,85 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class h5::StellarCollapseEos
+
+#pragma once
+
+#include <hdf5.h>
+#include <string>
+#include <vector>
+
+#include "DataStructures/BoostMultiArray.hpp" // IWYU pragma: keep
+#include "IO/H5/Object.hpp"
+#include "IO/H5/OpenGroup.hpp"
+
+// IWYU pragma: no_include <boost/multi_array.hpp>
+
+namespace h5 {
+/*!
+ * \ingroup HDF5Group
+ * \brief Reads in tabulated equation of state file from
+ * <a href="https:://stellarcollapse.org"> stellarcollapse.org </a>
```

Comment:
* This link doesn't work
* Use the `[]()` link form
* The text "stellarcollapse.org" is linked twice in this description and points at two different pages.  This is confusing.  Maybe just make "SRO ... State files" the link in the second one?